hardware_interface/src/mock_components/generic_system.cpp

```diff
@@ -723,11 +723,11 @@ void GenericSystem::initialize_storage_vectors(
           if (it2 != component.parameters.end())
           {
             states[index][i] = hardware_interface::stod(it2->second);
-            print_hint = true;
+            print_hint = false;
           }
           else
           {
-            print_hint = true;
+            print_hint = false;
           }
```

Comment:
just delete the else branch -> we don't want to print a warning if no initial value is given